public/static/locales/de/donate.json

```diff
@@ -47,7 +47,7 @@
   "seedingYourDonation": "Deine Spende wird bearbeitet",
   "pleaseDoNotCloseThisTab": "Bitte schließe diese Seite nicht!",
   "perTree": "/ Baum",
-  "stopTalkingStartPlanting": "Stop talking. Start planting.",
+  "stopTalkingStartPlanting": "Stop Talking. Start Planting.",
```

Comment:
Revert this changes, needs to be "Stop talking. Start planting." in all languages.